app/controllers/api/v1/webhooks_controller.rb

```diff
@@ -25,6 +26,15 @@ def twitter_events
     head :ok
   end
 
+  def agent_bot
```

Comment:
Please move this to agent_bot namespace. This controller is already doing a lot of things which have nothing in common. 

Instead of `webhooks/agent_bot`, this could be moved to `agent_bot/callback`

Reply:
Done